### ISSUE TITLE
Create filepath if it !exist for the <plugin_name>.socket at plugin runtime

### ISSUE THREAD

**Issue by nerdalert** [gopher-net/docker-ovs-plugin#7]
FATA[0000 socket filepath [ /usr/share/docker/plugins/  does not exist
The null is already in place. Just needs to match the cli parameter passed which is already parsed and split.

**Comment by runseb** [comment]
yep...sent a PR that adds this as a manual step in the README (for now).

**Comment by nerdalert** [comment]
Thanks @runseb , will update the binary with that patch and the bridge create patch shortly.
